app/views/admin/statistics_announcements/_form.html.erb

```diff
@@ -3,6 +3,7 @@
   <%= render "govuk_publishing_components/components/radio", {
     heading: "Statistics type (required)",
     name: "statistics_announcement[publication_type_id]",
+    id: "statistics_announcement_publication_type_id",
```

Comment:
As Nikin has said a bunch of these fields are missing errors being passed into the inputs https://github.com/alphagov/whitehall/blob/381805d6a1e6f72f8fb38872dc14df2e7177c128/app/views/admin/statistics_announcements/_form.html.erb#L3-L61

Apologies if i missed that in a previous PR!

Reply:
Yes, thank you, managed to overlook the component level error messages. Should be fully implemented now. ✅